### PR TITLE
Add various spelling corrections for words containing "meant"

### PR DESCRIPTION
**Pull request by cfi-gb**
See e.g.:
- https://github.com/search?q=meanting&type=code
- https://grep.app/search?q=meanting

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11083,6 +11083,10 @@ decrasing->decreasing, deceasing,
 decration->decoration
 decreace->decrease
 decreas->decrease
+decremeant->decrement
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:11086]: Add `decremeants`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -20181,7 +20185,10 @@ increament->increment
 increas->increase
 incredable->incredible
 incremantal->incremental
+incremeant->increment
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:20188]: Add `incremeants`?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:11086]: Thanks a lot for the suggestion. Added in 28d466750b178824e86dd2bdbf0ba82f69ab465b

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (cfi-gb) [reply codespell_lib/data/dictionary.txt:20189]: Thanks a lot for the suggestion. Added in 28d466750b178824e86dd2bdbf0ba82f69ab465b

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @cfi-gb.

**Pull request merged**